policy-controller/k8s/status/src/index.rs

```diff
@@ -233,3 +252,33 @@ pub(crate) fn make_patch(
     });
     k8s::Patch::Merge(value)
 }
+
+pub async fn create_lease_manager(client: k8s::Client) -> Result<kubert::lease::LeaseManager> {
+    let api = k8s::Api::namespaced(client, "linkerd");
+    match api
+        .create(
+            &Default::default(),
+            &k8s::Lease {
+                metadata: k8s::ObjectMeta {
+                    name: Some(STATUS_CONTROLLER_NAME.to_string()),
+                    namespace: Some("linkerd".to_string()),
+                    ..Default::default()
+                },
+                ..Default::default()
+            },
+        )
+        .await
+    {
+        // If the lease is created or already exists, start trying to claim
+        // it.
+        Ok(_) | Err(k8s::Error::Api(k8s::ErrorResponse { code: 409, .. })) => {}
```

Comment:
This won't actually attempt to claim it, right? We need kick off the manager with `spawn`

Reply:
That is correct. `LeaseManager::spawn` is the background task responsible for actually trying to claim the lease.